lib/board.rb

```diff
@@ -66,7 +65,7 @@ def valid_placement?(ship, coordinates)
   end
 
   def validated_placement
-    if (valid_coordinates_suite(@ship) && ship_overlap(@ship, @coordinates)) == true
+    if (valid_coordinates_suite(@ship) && ship_overlap(@ship, @coordinates))
```

Comment:
Awesome, you took off the unnecessary ==true statement? Love it! Way to think through our refactoring!

Reply:
Thank you!  I learn from the best!